core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -297,6 +298,16 @@ private GraphHopper setUnsafeMemory() {
         return this;
     }
 
+    public GraphHopper setProfiles(ProfileConfig... profiles) {
+        return setProfiles(Arrays.asList(profiles));
+    }
+
+    public GraphHopper setProfiles(List<ProfileConfig> profiles) {
```

Comment:
Some javadocs would be good here, also how to set the LM/CHProfileConfig. Or link to this PR.

Reply:
Yes will do as soon as we have agreed on a final design